testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/jpa/beanvalidation/beanvalidationtest/persistence.xml

```diff
@@ -9,6 +9,8 @@
         <jta-data-source>java:jboss/datasources/ExampleDS</jta-data-source>
         <properties>
             <property name="hibernate.hbm2ddl.auto" value="create-drop"/>
+            <property name="wildfly.jpa.twophasebootstrap" value="false"/>
+            <property name="jboss.as.jpa.classtransformer" value="true"/>
```

Comment:
What are these changes for?

Reply:
wildfly.jpa.twophasebootstrap switched from a two-phase bootstrap done with two different persistence unit services to a single phase bootstrap phase.

jboss.as.jpa.classtransformer is now defaulting to true currently but we have changed that setting a few times so I added the property to force bytecode enhancement to always run for this test.  As mentioned above, we pass with and without these changes.

Reply:
I removed these test changes in case they help the tests pass somehow.